Sources/BrowserServicesKit/Suggestions/SuggestionProcessing.swift

```diff
@@ -80,7 +80,6 @@ final class SuggestionProcessing {
     // MARK: - DuckDuckGo Suggestions
 
     private func duckDuckGoSuggestions(from result: APIResult?) throws -> [Suggestion]? {
-        // TODO add query as duckduckgo suggestion
```

Comment:
Should I be concerned this never got done?

Reply:
I had the same thought, I checked when it was added and it looked like it had existed for 2+ years. @tomasstrba do you know if this is still applicable?

Reply:
Created an Asana task to track this so that the PR isn't blocked

Reply:
This can be removed (even from Asana) because it's just a forgotten comment. The point behind it was to create a suggestion from the query like the Firefox does but we don't need that.

<img width="995" alt="Screenshot 2023-08-18 at 8 57 31 AM" src="https://github.com/duckduckgo/BrowserServicesKit/assets/57389842/17b58d00-f558-4c97-bbfa-f3a33f29e528">